torch/_inductor/codegen/cuda/cutlass_epilogue_gen.py

```diff
@@ -0,0 +1,337 @@
+from typing import Dict, List
+from unittest.mock import patch
+
+import sympy
+
+import torch._inductor.virtualized as virtualized
+from torch._inductor.ir import ComputedBuffer, FlexibleLayout, IRNode, Pointwise
+from torch._inductor.utils import IndentedBuffer, sympy_str
+
+
+def _arg_str(a):
+    if isinstance(a, sympy.Expr):
+        return "@sympy_expr('" + sympy_str(a) + "')"
```

Comment:
Why the `@` here but not in the case below?

Reply:
it's used as a magic string that I search for later. If this magic string appears in the generated code, a sympy expression would have to be evaluated, which is not yet supported here, so I need to err out in that case ( in that case, a clean fallback is to simply reject the fusion during can_fuse )

Reply:
Why not just throw an error here?

Reply:
reBecause it's not an error if the string is created, only if it becomes part of the generated code, which depends on how the return value is used.

Reply:
Add a comment explaining this.  Also suggest not making it look like a decorator (pick syntax that doesn't mean something in python).